### ISSUE TITLE
Cannot run `ExpReport()` on `attitude` dataset

### ISSUE THREAD

**Issue by fkohrt** [daya6489/SmartEDA#10]
If I run:

```r
SmartEDA::ExpReport(attitude, op_file = "SmartEDA.html")
```

I get the following error:

```
Error in paged_table(et1) : 
  Only data frames can be used to create a paged_table.
```

<details>
<summary><code>sessionInfo()</code></summary>

```
R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.6 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/libopenblasp-r0.2.20.so

locale:
 [1] LC_CTYPE=en_US.UTF-8      
 [2] LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8   
 [6] LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8      
 [8] LC_NAME=C                 
 [9] LC_ADDRESS=C              
[10] LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8
[12] LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils    
[5] datasets  methods   base     

other attached packages:
[1] ggplot2_3.4.0  gridExtra_2.3  scales_1.2.1  
[4] knitr_1.41     SmartEDA_0.3.8 rmarkdown_2.18

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.0   xfun_0.35         
 [3] bslib_0.4.1        colorDF_0.1.7     
 [5] lpSolve_5.6.17     purrr_0.3.5       
 [7] colorspace_2.0-3   vctrs_0.5.1       
 [9] generics_0.1.3     htmltools_0.5.3   
[11] yaml_2.3.6         utf8_1.2.2        
[13] rlang_1.0.6        pillar_1.8.1      
[15] jquerylib_0.1.4    glue_1.6.2        
[17] withr_2.5.0        DBI_1.1.3         
[19] RColorBrewer_1.1-3 lifecycle_1.0.3   
[21] plyr_1.8.8         ISLR_1.4          
[23] stringr_1.4.1      munsell_0.5.0     
[25] gtable_0.3.1       evaluate_0.18     
[27] labeling_0.4.2     GGally_2.1.2      
[29] fastmap_1.1.0      sampling_2.9      
[31] fansi_1.0.3        highr_0.9         
[33] Rcpp_1.0.9         cachem_1.0.6      
[35] jsonlite_1.8.3     farver_2.1.1      
[37] digest_0.6.30      stringi_1.7.8     
[39] dplyr_1.0.10       grid_4.2.1        
[41] cli_3.4.1          tools_4.2.1       
[43] magrittr_2.0.3     sass_0.4.4        
[45] tibble_3.1.8       crayon_1.5.2      
[47] pkgconfig_2.0.3    MASS_7.3-58.1     
[49] data.table_1.14.6  assertthat_0.2.1  
[51] reshape_0.8.9      rstudioapi_0.14   
[53] R6_2.5.1           compiler_4.2.1    
```

</details>

**Comment by daya6489** [comment]
Thanks @fkohrt for raising these issues. It has been fixed in 0.3.9 version. Please update SmartEDA library and check the same...

**Comment by fkohrt** [comment]
Yes, now it works for me, thanks!
